pages/1.12/release-notes/1.12.2/index.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.2
+title: Release Notes for 1.12.2
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.12.2
+---
+
+DC/OS Version 1.12.2 was released on February 7, 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.12.2/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise [/button]
+
+DC/OS 1.12.2 includes the following components:
+- Apache Mesos 1.7.x [change log](https://github.com/apache/mesos/blob/b97f0ba29d40a279dec00ffe51512e3b5a146049/CHANGELOG).
```

Comment:
link should be: https://github.com/apache/mesos/blob/4f21147a9334ef9ed84dcf11742ce448062f3ec/CHANGELOG
SHA seen here: https://github.com/dcos/dcos/blob/1.12.2/packages/mesos/buildinfo.json#L11

Reply:
Changed the link.
Is there something I need to do with the SHA json link? I haven'tseen that before.

Reply:
@lsgunnlsgunn I put those there to show how I found the correct links so that docs team can find them in the future

Reply:
cool--thanks for providing that link. I might ask you to verify I grabbed the correct info next time but it is great to know how to get to it myself.